Sources/StreamChat/Query/Pagination.swift

```diff
@@ -25,7 +25,7 @@ public struct Pagination: Encodable, Equatable {
     /// An offset.
     public let offset: Int
     
-    private enum CodingKeys: String, CodingKey {
+    public enum CodingKeys: String, CodingKey {
```

Comment:
Not sure if we should expose these as public, it might be trickier to change them in the future. Maybe just use string value where needed in tests?